src/BatteryStateOfChargeService.h

```diff
@@ -38,12 +38,21 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     double totalAmpHoursUsed() const;
     bool isCharging() const;
     QTime timeWhenChargedOrDepleted() const;
+    QTime time;
```

Comment:
There are very few cases when a member is going to be public, this QTime time should be put as a private member, and if it is really necessary that it is accessed, a getter function should be used instead.